laghos_solver.cpp

```diff
@@ -1230,12 +1230,12 @@ void QKernel(const int NE, const int NQ,
             MFEM_FOREACH_THREAD(qy,y,Q1D)
             {
                QUpdateBody<DIM>(NE, e, NQ, qx + qy * Q1D,
-               use_viscosity, use_vorticity, h0, h1order, cfl, infinity,
-               Jinv, stress, sgrad_v, eig_val_data, eig_vec_data,
-               compr_dir, Jpi, ph_dir, stressJiT,
-               d_gamma, d_weights, d_Jacobians, d_rho0DetJ0w,
-               d_e_quads, d_grad_v_ext, d_Jac0inv,
-               d_dt_est, d_stressJinvT);
+                                use_viscosity, use_vorticity, h0, h1order, cfl, infinity,
+                                Jinv, stress, sgrad_v, eig_val_data, eig_vec_data,
+                                compr_dir, Jpi, ph_dir, stressJiT,
+                                d_gamma, d_weights, d_Jacobians, d_rho0DetJ0w,
+                                d_e_quads, d_grad_v_ext, d_Jac0inv,
+                                d_dt_est, d_stressJinvT);
```

Comment:
@tzanio Are you happy with this? Because this looks like what I got from Clang format...

Reply:
yes, this is OK